src/pages/index.js

```diff
@@ -75,10 +129,21 @@ addCardButton.addEventListener("click", () => {
 const editProfileFormValidator = new FormValidator(validationSettings, profileEditForm);
```

Comment:
Please, delete `this._toggleButtonState();` from line 77 in `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.